### DISCUSSION TITLE
Notes Nov 10th, 2021

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre#3] [Technical Steering Committee Meetings]
### MapLibre GL JS Technical Questions

[MaxD/Toursprung] Present the current status of 3D, discuss the open issues, create a timetable for their fixes and settle on a merge roadmap.
https://static.maptoolkit.net/maplibre/terrain.html 
https://github.com/maplibre/maplibre-gl-js/pull/165 
Tech docs @max  + separate meeting to be organized by @helge
Invitees:  @roblabs 

[Harel M] Named exports and typescript import issues
https://github.com/maplibre/maplibre-gl-js/pull/623 
Blocker currently - ensuring back compatibility with old typing for TypeScript - Tree-shaking would be a big benefit of named exports

### Organizational Questions

Minutes to be put into Discussion in “maplibre/maplibre” @klokan
Disable “wiki” @klokan
News on the web @oliver
Process of adding people to sub-projects
Can be granted to “active contributors” (~2 PRs), preferably on individual repos
Revisit group rights (github+npm) @yuri

### Attendees
@klokan - Petr Pridal
@prozessor13 (MaxD)
@muesliq (HelgeF)
@HarelM - Harel Mazor
@marcelnormann - Marcel Normann
@Joxit - Jones Magloire
@nyurik - Yuri Astrakhan
@roblabs — Rob Chohan
@viktorgt - Viktor Gottfried
@lseelenbinder - Luke Seelenbinder
@wipfli - Oliver Wipfli

### Next meeting

Dec 8th, 2021 at 20:00 GMT+1.

Feel free to share your thoughts below.

**Comment by nnhubbard** [comment]
Will and of the 3D feature be added to `MapLibre GL Native` so that we can use it on iOS?

**Comment by prozessor13** [comment]
Hello,

I will not do the implementation, but i'll be glad to share all my knowlege
collected during the 3D implementation in mapbox-gl-js.

Regards. Max.
